src/groundStation/groundStation.py

```diff
@@ -397,15 +397,22 @@ def getOptions(self):
             type=str,
             default="EX2",
             help='Satellite parameter for automatic programs (e.g FTP)')
+
+        self.parser.add_argument(
```

Comment:
Should have a more descriptive name than just --file

Reply:
This change hasn't made it to github

Reply:
-S used instead